nipype/interfaces/fsl/tests/test_model.py

```diff
@@ -6,23 +6,23 @@
 import pytest
 import nipype.interfaces.fsl.model as fsl
 from nipype.interfaces.fsl import no_fsl
+from pathlib import Path
 
 
 @pytest.mark.skipif(no_fsl(), reason="fsl is not installed")
 def test_MultipleRegressDesign(tmpdir):
-    tmpdir.chdir()
     foo = fsl.MultipleRegressDesign()
     foo.inputs.regressors = dict(
         voice_stenght=[1, 1, 1], age=[0.2, 0.4, 0.5], BMI=[1, -1, 2]
     )
     con1 = ["voice_and_age", "T", ["age", "voice_stenght"], [0.5, 0.5]]
     con2 = ["just_BMI", "T", ["BMI"], [1]]
-    foo.inputs.contrasts = [con1, con2, ["con3", "F", [con1, con2]]]
+    foo.inputs.contrasts = [con1, con2, ["con3", "F", [con1, con2]], ["con4", "F", [con2]]]
     res = foo.run()
 
     for ii in ["mat", "con", "fts", "grp"]:
         assert (
-            getattr(res.outputs, "design_" + ii) == tmpdir.join("design." + ii).strpath
+            os.path.exists(eval('res.outputs.design_'+ii))
         )
```

Comment:
I'm not a fan of `eval`. It's a big heavy hammer for pulling a variable out of the local ether. I would rather do something like:

```Python
outputs = res.outputs.get_traitsfree()  # This fetches `outputs` contents as a dict.
for ftype in ["mat", "con", "fts", "grp"]:
    assert Path(outputs["design_" + ftype]).exists()
```